RUN_PROTOCOL.md

```
@@ -0,0 +1,222 @@
+node -e "import('./scripts/load-browser-config.mjs').then(m=>m.loadBrowserConfig()).then(c=>console.log('OK', c.preferred, '| extension_autofill:', c.extension_autofill))"
+```
+
+Expected output: `OK firefox | extension_autofill: true`
+If you see a `BrowserConfigError`, fix the field it names.
+
+**How it works:** With `extension_autofill: true`, auto-submit launches Firefox using your real
+profile, navigates to the job URL, then waits 5 seconds for SpeedyApply to detect the form and
+autofill it. Our built-in field selectors (Greenhouse/Lever/Workday) are skipped entirely.
+
+To opt out of extension autofill on a single run:
+```powershell
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)